.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "es2015",
```

Comment:
AFAIK ES6 == ES2015, so this is just E6, but yeah, this might make it harder to import in other repos. I'd like to first try importing `clever/components` in another repo before settling for less than ES6.